src/types/traces.ts

```diff
@@ -0,0 +1,90 @@
+import { Color } from "./color";
+
+// Colour chart for traces
+
+const TRACE_COLOR_ARRAY: { [key: number]: Color } = {
+  0: new Color("rgb(21, 21, 196)"), // blue
+  1: new Color("rgb(242, 26, 26)"), // red
+  2: new Color("rgb(33, 179, 33)"), // green
+  3: new Color("rgb(0, 0, 0)"), // black
+  4: new Color("rgb(128, 0, 255)"), // violet
+  5: new Color("rgb(255, 170, 0)"), // (darkish) yellow
+  6: new Color("rgb(255, 0, 240)"), // pink
+  7: new Color("rgb(243, 132, 132)"), // peachy
+  8: new Color("rgb(0, 255, 11)"), // neon green
+  9: new Color("rgb(0, 214, 255)"), // neon blue
+  10: new Color("rgb(114, 40, 3)"), // brown
+  11: new Color("rgb(219, 128, 4)"), // orange
+  12: new Color("rgb(21, 21, 196)"), // Repeat colours until 20
+  13: new Color("rgb(242, 26, 26)"),
+  14: new Color("rgb(33, 179, 33)"),
+  15: new Color("rgb(0, 0, 0)"),
+  16: new Color("rgb(128, 0, 255)"),
+  17: new Color("rgb(255, 170, 0)"),
+  18: new Color("rgb(255, 0, 240)"),
+  19: new Color("rgb(243, 132, 132)")
```

Comment:
Does this mean that we will limit the number of traces on a plot to 20? I see that CS-Studio also limits to 20 traces. What would happen if there were more - do they just get ignored?

Reply:
I based this off CS-Studio's limit of 20, although I don't know if it's something we want to continue to enforce or allow for more traces? You raise a good point I hadn't considered though, I don't know what CSS did if too many traces were provided. In this case, I think having more than 20 traces would throw an index out of range error when it attempts to allocate a colour, which it definitely shouldn't do